src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
I think it might be better if the String had a more descriptive name, perhaps something like "date" or "deadline"?